README.md

```diff
@@ -1,3 +1,4 @@
+[![codecov](https://codecov.io/gh/Kalafut-organization/elephant_vending_machine_frontend/branch/master/graph/badge.svg)](https://codecov.io/gh/Kalafut-organization/elephant_vending_machine_frontend)
```

Comment:
```suggestion
![Github Actions for Elephant Vending Machine - Front-end](https://github.com/Kalafut-organization/elephant_vending_machine_frontend/workflows/Github%20Actions%20for%20Elephant%20Vending%20Machine%20-%20Front-end/badge.svg)
[![MIT license](https://github.com/Kalafut-organization/elephant_vending_machine_frontend/blob/master/LICENSE.md)
[![codecov](https://codecov.io/gh/Kalafut-organization/elephant_vending_machine_frontend/branch/master/graph/badge.svg)](https://codecov.io/gh/Kalafut-organization/elephant_vending_machine_frontend)
```

Might as well go ahead and add the other relevant badges like we have for the backend repo.